terraform/aws/modules/atlantis/atlantis.tf

```diff
@@ -69,3 +69,17 @@ resource "helm_release" "atlantis" {
   timeout = 1200
 
 }
+
+resource "aws_instance" "at" {
+  ami               = var.ami_id
```

Comment:
@stafot Considering that this is a single instance, how do we handle instance termination and lost of Atlantis installation?

Reply:
@stylianosrigas my understanding is to treat it as immutable infra so create a new ami from the existing which already has what we need and every time a disaster happens redeploy the same(keeping it up to date and take constant snapshots too).

Reply:
We could have an ASG with 1/1/1 so this can be automated

Reply:
@angeloskyratzakos make sense